XMonad/Actions/Navigation2D.hs

```diff
@@ -824,7 +824,7 @@ doSideNavigationWithBias bias dir (cur, rect)
     rHalfPiCC r = SideRect (-y2 r) (-y1 r) (x1 r) (x2 r)
 
     -- Apply the above function until d becomes synonymous with R (wolog).
-    rotateToR d = let (_, _:l) = break (d ==) [U, L, D, R]
+    rotateToR d = let (_, nonEmpty -> _ :| l) = break (d ==) [U, L, D, R]
```

Comment:
This works, but perhaps we could just do `drop 1 $ dropWhile (d /=) [U, L, D, R]` to get rid of `nonEmpty` entirely here

Reply:
I ended up rewriting it:
```haskell
rotateToR d = fromJust . lookup d . zip [R, D, L, U] . iterate rHalfPiCC
```
Like elsewhere, I think a verbose blowup is better than silent misbehaviour.

Reply:
3bfbd2239d543a6419227fe38ce79678ea442d2a